packages/mds-stream/kafka/stream-consumer.ts

```diff
@@ -121,6 +126,7 @@ const createStreamConsumer = async (
     await Promise.all(asArray(topics).map(topic => consumer.subscribe({ topic, fromBeginning })))
 
     registerLivelinessHandling(consumer, healthStatus)
+    delete healthStatus.components[preConnectComponentId]
```

Comment:
I don't _love_ using `delete` (I generally try to avoid), but the alternatives are somewhat cumbersome.